src/backend/dx11/src/lib.rs

```diff
@@ -1363,7 +1530,7 @@ pub struct DescriptorSet {
     #[derivative(Debug="ignore")]
     srv_handles: RefCell<Vec<(u32, ComPtr<d3d11::ID3D11ShaderResourceView>)>>,
     #[derivative(Debug="ignore")]
-    cbv_handles: RefCell<Vec<(u32, ComPtr<d3d11::ID3D11Buffer>)>>,
+    cbv_handles: RefCell<Vec<(u32, *mut d3d11::ID3D11Buffer)>>,
```

Comment:
why the raw pointer here?

Reply:
It was to avoid having a `Drop` impl for the `Buffer` struct, since portability [creates a zeroed struct](https://github.com/gfx-rs/portability/blob/master/libportability-gfx/src/impls.rs#L1145-L1147) which gets dropped (and tries to free null ComPtr)

Reply:
looks like a bug in `portability` than that we need to fix